cmd/cluster-node-tuning-operator/main.go

```diff
@@ -177,6 +183,65 @@ func tunedOperandRun() {
 	tuned.Run(stopCh, &boolVersion, version.Version)
 }
 
+// Uninstall PAO OLM operator.
+// This should apply only from version 4.11
```

Comment:
While this comment is clear to us now, it may not be to someone else in the future.  Perhaps saying something about PAO shipping a core operator from 4.11 and no longer as OLM operator and that's why we are removing it?  I wonder if we should also add something like "Remove this code in OCP version 4.XY" if we know what XY should be, or at least "Remove this code in the future.".

Reply:
Note taken.  We can add  "Remove this code in OCP version 4.XY" if we can tell for sure that the upgrade path will not come from 4.10 (and below) to the version in hand (be it 4.11, 4.12 , ...)

Reply:
comment updated.